data/src/test/java/org/oppia/android/data/backends/gae/api/FeedbackReportingServiceTest.kt

```diff
@@ -29,7 +28,6 @@ class FeedbackReportingServiceTest {
   @Before
   fun setUp() {
     val client = OkHttpClient.Builder()
-    client.addInterceptor(NetworkInterceptor())
```

Comment:
Does `FeedbackReportingService` do not need `JsonPrefixNetworkInterceptor ` ?

Reply:
The service currently doesn't receive anything in the response, so I don't believe so since the interceptor only handles responses.

Reply:
Ack. Thanks.